int32 xian_dj_robotgo1_params_server_heart_beat 
int32 xian_dj_robotgo1_error_monitor_heart_beat 
int32 xian_dj_robotgo1_back_end_heart_beat 
int32 xian_dj_robotgo1_state_display_heart_beat 
int32 xian_dj_robotgo1_switch_mode_heart_beat 
int32 xian_dj_robotgo1_control_heart_beat 
int32 xian_dj_robotgo1_lighter_displayer_heart_beat 
int32 xian_dj_robotgo1_tele_op_heart_beat 
int32 xian_dj_robotgo1_local_controller_heart_beat 
int32 xian_dj_robotgo1_battery_level_heart_beat 
int32 xian_dj_robotgo1_auto_mode 
int32 xian_dj_robotgo1_tele_op_mode 
int32 xian_dj_robotgo1_m_system_mode 
int32 xian_dj_robotgo1_s_system_mode 
string xian_dj_robotgo1_battery_level 
int32 xian_dj_robotgo1_display_mode 
int32 xian_dj_robotgo1_red_ligher_cmd 
int32 xian_dj_robotgo1_green_ligher_cmd 
int32 xian_dj_robotgo1_yellow_ligher_cmd 
string xian_dj_robotgo1_displayer_cmd 
int32 xian_dj_local_controller_left_cmd 
int32 xian_dj_local_controller_right_cmd 
int32 xian_dj_local_controller_up_cmd 
int32 xian_dj_local_controller_down_cmd
int32 xian_dj_local_controller_x_cmd 
int32 xian_dj_local_controller_b_cmd 
int32 xian_dj_local_controller_y_cmd 
int32 xian_dj_local_controller_a_cmd 

float32 xian_dj_local_controller_left_rocker_x_cmd 
float32 xian_dj_local_controller_left_rocker_y_cmd 
float32 xian_dj_local_controller_right_rocker_x_cmd 
float32 xian_dj_local_controller_right_rocker_y_cmd 

int32 xian_dj_local_controller_r1_cmd 
int32 xian_dj_local_controller_r2_cmd 
int32 xian_dj_local_controller_l1_cmd 
int32 xian_dj_local_controller_l2_cmd 
int32 xian_dj_robotgo1_params_server_error 
int32 xian_dj_robotgo1_state_display_error 
int32 xian_dj_robotgo1_switch_mode_error 
int32 xian_dj_robotgo1_back_end_error 
int32 xian_dj_robotgo1_control_error 
int32 xian_dj_robotgo1_lighter_displayer_error  
int32 xian_dj_robotgo1_error_code 
int32 xian_dj_robotgo1_heat_beat_error_code